project/Dependencies.scala

```diff
@@ -34,8 +34,7 @@ object Dependencies {
 
     val r2dbcSpi = "io.r2dbc" % "r2dbc-spi" % "1.0.0.RELEASE"
     val r2dbcPool = "io.r2dbc" % "r2dbc-pool" % "1.0.2.RELEASE"
-    val r2dbcPostgres = Seq(
-      "org.postgresql" % "r2dbc-postgresql" % "1.0.7.RELEASE")
+    val r2dbcPostgres = "org.postgresql" % "r2dbc-postgresql" % "1.0.7.RELEASE"
```

Comment:
This is ok, but better we later includes it in release note too

Reply:
yes - this will definitely be in the release notes